src/test/java/plugins/KerberosSsoTest.java

```diff
@@ -301,7 +300,9 @@ private User setupRealmUser() {
         JenkinsDatabaseSecurityRealm realm = sc.useRealm(JenkinsDatabaseSecurityRealm.class);
         sc.save();
         // The password needs to be the same as in kerberos
-        return realm.signup().password("ATH").fullname("Full Name").signup("user");
+        return realm.signup().password("ATH").fullname("Full Name")
+                .email("ath@ath.com")
```

Comment:
IIRC this require mailer plugin to be installed, which it might not be on Jenkins 2.x. Is the field required when it is installed? I presume you are hitting this on Jenkins 1.X where it is bundled. In that case, we need to require mailer explicitly.